src/tests/ftest/util/test_utils_base.py

```diff
@@ -160,18 +160,18 @@ def _check_info(self, check_list):
             # Determine which comparison to utilize for this check
             compare = ("==", lambda x, y: x == y, "does not match")
             if isinstance(expect, str):
-                comparisons = {
-                    "<": (lambda x, y: x < y, "is too large"),
-                    ">": (lambda x, y: x > y, "is too small"),
-                    "<=": (
-                        lambda x, y: x <= y, "is too large or does not match"),
-                    ">=": (
-                        lambda x, y: x >= y, "is too small or does not match"),
-                }
-                for key, val in list(comparisons.items()):
+                comparisons = [
```

Comment:
Why was this file modified?  It doesn't seem to be needed as it just redefines a python dictionary as a list of lists.

Reply:
Ah I see.  Thanks.